stable/fairwinds-insights/templates/ingress.yaml

```diff
@@ -1,12 +1,12 @@
 {{- if .Values.ingress.enabled }}
 {{- $fullName := include "fairwinds-insights.fullname" . -}}
+{{- if not .Values.ingress.separate }}
```

Comment:
is `.Values.ingress.separate` added in values.yaml? what is the default value and i can not seem to find it in documentation[README].

Reply:
oh whoops - will add docs in a separate PR